gef-ui/server/utils/MOCKS/mock_applications.js

```diff
@@ -138,6 +138,282 @@ const MOCK_AIN_APPLICATION = {
   },
 };
 
+const MOCK_AIN_APPLICATION_CHECKER = {
```

Comment:
can we use `chance.js` to generate random values for some of these properties?

Reply:
done